### ISSUE TITLE
Upgrade Nightwatch and Chrome Driver to latest version in content-build

### ISSUE THREAD

**Issue by ddzz** [department-of-veterans-affairs/va.gov-team#26220]
## Issue Description
This task is to upgrade Nightwatch and Chrome Driver to the latest versions in content-build. This may address some issues with stability of accessibility tests.

This was done recently in vets-website: https://github.com/department-of-veterans-affairs/vets-website/pull/17560

## Tasks
- [x] Update Nightwatch in content-build
- [x] Update Chrome Driver in content-build
- [ ] Verify that Nightwatch upgrade didn't break the daily a11y scans in content-build

## Tasks
- [ ] Updates are completed and working as intended and have been verified
- [ ] Peter/Darius have approved

**Comment by CBonade** [comment]
A quick update on this as it was put on hold waiting on the a11y scan to be tidied up.  Been trying to help Tim out with this today tinkering with different reasons that the a11y scan either:

1) Fails when it should pass
2) Fails but doesn't stop the test, and marks it as a success.

Until we can verify that the a11y scan has no other problems that have nothing to do with these upgrades, testing these upgrades won't be possible.  The current status of our latest timeout adjustment is 2 of the tests (one should have passed,and one was a forced failure) both behaved appropriately, so we're hopeful that we're close to a solution.  The next step is a full site run with an intentional failure, which will take multiple hours.  Will update this ticket again when progress moves forward.

**Comment by CBonade** [comment]
This was requested to be put on hold awaiting the deactivation of the Jenkins a11y scan when it's fully moved over to GHA.  That seems to now be dependent on the Cypress a11y scan conversion from Nightwatch being completed. WIP will keep this updated as it moves along (the PR still exists here: https://github.com/department-of-veterans-affairs/content-build/pull/266 , originally pushed 20 days ago).

**Comment by CBonade** [comment]
Still on hold here pending a11y test conversion to Cypress and/or fixing in Jenkins with Nightwatch

**Comment by CBonade** [comment]
Went ahead and moved this back to the backlog as it will need reevaluating once the a11y test situation is sorted.

**Comment by CBonade** [comment]
Weekly Update: Still holding waiting on resolution to a11y test issues

**Comment by CBonade** [comment]
With the addition of the Cypress A11y test in `content-build`, instead of managing Nightwatch we are going to move towards deprecating Nightwatch in `content-build` instead.  Moving this to review for closing.  Ticket to deprecate nightwatch in content-build here: https://app.zenhub.com/workspaces/vsp-5cedc9cce6e3335dc5a49fc4/issues/department-of-veterans-affairs/va.gov-team/29611

**Comment by CBonade** [comment]
This PR has been closed out as Nightwatch is on its way out of content-build. Ready to be closed.

**Comment by pjhill** [comment]
This effort was OBE due to removal of Nightwatch from `content-build`.
